test/chainsaw/testmetrics/dynatrace-provider/00-assert.yaml

```diff
@@ -0,0 +1,9 @@
+
+apiVersion: metrics.keptn.sh/v1beta1
```

Comment:
please make sure all namespaces and all resource names between the installation and assertion match otherwise the test won't work

Reply:
Resolved!